
# HG changeset patch
# User psadhukhan
# Date 1445944584 -10800
# Node ID dfeaf40df80032570558c031f37f6630dc729e3f
# Parent  8e4b93a5a036bec10caeed3c1063b63e5ca571b1
8139803: Fix for 8132985 breaks OpenJDK build on windows.
Reviewed-by: serb, simonis

diff -r 8e4b93a5a036 -r dfeaf40df800 jdk/src/share/native/sun/font/freetypeScaler.c
--- openjdk/jdk/src/share/native/sun/font/freetypeScaler.c	Tue Oct 27 14:14:08 2015 +0300
+++ openjdk/jdk/src/share/native/sun/font/freetypeScaler.c	Tue Oct 27 14:16:24 2015 +0300
@@ -103,7 +103,6 @@
 }
 
 static void freeNativeResources(JNIEnv *env, FTScalerInfo* scalerInfo) {
-    void *stream;
 
     if (scalerInfo == NULL)
         return;

